theme/templates/accounts/account_password_reset.html

```diff
@@ -25,16 +25,15 @@
             <div class="row">
                 {% include "includes/checkbox_field.html" with field=form.email  field_classes="col-md-12" %}
             </div>
-
+            <p class="text-muted">Enter your username or email address and you'll receive an email with a link you need to click, in order to log in and change your password.</p>
```

Comment:
This is more of an English suggestion, but I don't think you need the comma in the above sentence ("...you need to click in order to...").